### PR TITLE
doc/ko: Update CONTRIBUTING.md in Korean

### PR DESCRIPTION
**Pull request by jigsaw-0**
CONTRIBUTING.md file is translated into Korean.
------------------------------------------------------

Few notices in this translation:
* Rather than a 1:1 translation between English and Korean, I focused on being as natural as possible when reading in Korean.
* Due to some of awkwardness, I used polite words in translation. 

Since other doc (.md) files were not translated using polite words, if you think this CONTRIBUTING.md file needs to be changed for a consistency with other documents, please let me know.

### CODE REVIEW

File: doc/ko/CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+패치에 서명하기
+---------------
+
+sign-off (서명)은 패치에 대한 설명 마지막 부분에 한줄로
+패치를 자신이 직접 작성했고, 오픈소스 패치로 배포할 권리가 있다는 사실을 알려주는 것입니다.
+sign-off에 대한 규칙은 [다음 항목](https://developercertificate.org/)들을 증명할 수 있는 것으로 충분합니다:
+
+		Developer's Certificate of Origin 1.1
+
+		본 프로젝트에 기여함으로써, 다음을 증명합니다:
```
Comment (namhyung) [inline doc/ko/CONTRIBUTING.md:81]: 복잡한 내용이라 번역하기 힘드셨을것 같습니다. 제 생각에는 기여자의 입장에서 본인이 (a), (b), (c) 조건 중의 하나를 만족하는지와 (d) 에 동의하는지에 대한 부분이 좀더 명확하게 표현되면 좋을것 같습니다.

**Review by namhyung — COMMENTED** [review]
Thanks for your good translation!

File: doc/ko/CONTRIBUTING.md [hunk 2]
```diff
@@ -89,14 +89,15 @@ sign-off에 대한 규칙은 [다음 항목](https://developercertificate.org/)
 			파일에 명시된 바에 따라, 기여자 본인은 동일한 오픈소스 라이센스에 기반하여
 			이전 작업들을 전체 또는 일부 수정하여 제출할 권리가 있습니다.
 
-		(c) 본 프로젝트에 대한 기여는 상기 (a), (b) 또는 (c)가 증명된 사람으로부터 직접 제공받았으며,
-			본인은 이를 수정하지 않았습니다.
+		(c) 본 프로젝트에 대한 기여는 상기 (a), (b) 항목 또는 (c) 항목에 해당되는 이로부터 직접 제공받았으며,
```
Comment (namhyung) [inline doc/ko/CONTRIBUTING.md:92]: (a), (b), (c) 중 하나의 항목에 해당하는 이로부터 ...

**Review by namhyung — COMMENTED** [review]
Looks ok, can you please squash the commits into one?

**Comment by paranlee** [comment]
@jigsaw-0 If you need squash to one commit, contact me to discord.
I will review process. :)
 - https://docs.google.com/document/d/1gCrBig2bKpMyX-bd_JRbphJHVZp2O7ruOK9VC31ktYU/edit#heading=h.t77h8nf1q3zk

**Review by namhyung — APPROVED** [review]
LGTM

**Pull request merged**